roles/junit2json/tests/test.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: "Test redhatci.ocp.junit2json role :: simple input"
+  hosts: localhost
+  tasks:
+    - name: Test role redhatci.ocp.junit2json
+      ansible.builtin.include_role:
+        name: redhatci.ocp.junit2json
+      vars:
+        junit2json_input_reports_list:
+          - "{{ role_path }}/../../tests/unit/data/test_junit2obj_simple_input.xml"
```

Comment:
- is `role_path` var a requirement for this test?
- where is this file located? this `tests/unit/data...` is not present in this change

Reply:
`role_path` is a special built-in variable, I don't need to define it.
The data `tests/unit/data` is under the collection root folder.
(2 folders up `role_path`)